shell/browser/api/electron_api_utility_process.cc

```diff
@@ -265,28 +284,46 @@ void UtilityProcessWrapper::PostMessage(gin::Arguments* args) {
     return;
 
   blink::TransferableMessage transferable_message;
+
+  auto* isolate = args->isolate();
```

Comment:
In general only use `auto` when the full type is given on the same line, please!

```suggestion
  v8::Isolate* isolate = args->isolate();
```